lab2.js

```diff
@@ -39,32 +36,68 @@ function assert(expression, failureMessage) {
         1          |       2/hour
         2          |       3/hour
         3          |       4/hour
-
- TODO: First, make a constructor function, called Blob, that makes blobs.
-
- TODO: Next, create an instance of Blob named blob.
-
- TODO: Then, use a loop to calculate how long it took the blob to finish
- with Dowington.
 */
+// TODO: First, make a constructor function, called Blob, that makes blobs.
+function Blob () {
+  this.consumptionRate = 1;
+  this.consumed = 0;
+}
 
-var hoursSpentInDowington; // TODO: assign me the value of the
-                           // above calculation
+// TODO: Next, create an instance of Blob named blob.
+var blob = new Blob();
+// TODO: Then, use a loop to calculate how long it took the blob to finish
+// with Dowington.
+
+var dowingtonPopulation = 1000;
+function consumeDowington() {
+  var hours = 0;
+  while (blob.consumed < dowingtonPopulation) {
+    hours += (1 / blob.consumptionRate);
```

Comment:
there's a couple ways you could do the rate. I'm accepting both, since I know the directions weren't all that clear.